templates/_helpers.tpl

```diff
@@ -35,6 +35,14 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Define the image for a container.
+*/}}
+{{- define "amrc-connectivity-stack.image" -}}
+image: {{ .image.registry }}/{{ .image.repository }}:{{ .image.tag }}
+imagePullPolicy: {{ .image.pullPolicy }}
+{{- end }}
+
```

Comment:
What does this do?

Reply:
This block is used in `templates/mqtt/mqtt.yaml` to pull in the image references. Factoring it out into a block avoids repeating the logic for the registry/repo/tag/pullPolicy every time.